sources/cfsignal/README.tpl

```diff
@@ -0,0 +1,9 @@
+# {{crate}}
```

Comment:
It looks like the actual `README.md` is missing.  Can you add that to your commit as well?